tests/acceptance/test_onboarding.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+from sentry.models import Project
+from sentry.testutils import AcceptanceTestCase
+
+
+class OrganizationOnboardingTest(AcceptanceTestCase):
+    def setUp(self):
+        super(OrganizationOnboardingTest, self).setUp()
+        self.user = self.create_user('foo@example.com')
+        self.org = self.create_organization(
+            name='Rowdy Tiger',
+            owner=self.user,
+        )
+        self.team = self.create_team(organization=self.org, name='Mariachi Band')
+        self.member = self.create_member(
+            user=None,
+            email='bar@example.com',
+            organization=self.org,
+            role='owner',
+            teams=[self.team],
+        )
+        self.login_as(self.user)
+        self.path = '/onboarding/%s/' % self.org.slug
+
+    def test_onboarding(self):
+        self.browser.get(self.path)
+        self.browser.wait_until('.onboarding-container')
+        self.browser.wait_until_not('.loading-indicator')
+        self.browser.snapshot(name='onboarding-choose-platform')
```

Comment:
we already snapshot this page as `organization onboarding`, you should remove that or test or just use this test to verify something else about the dom (we don't want to snapshot the same thing twice in percy or use it ensure logic)
https://github.com/getsentry/sentry/blob/master/tests/acceptance/test_organization_onboarding.py#L22

Reply:
It looks like that test was created after my PR? It feels like it's sort of a subset of this test (doesn't do actual project creation), I'd advise we delete that one.

Reply:
Fixed.